source/MicroBitLog.cpp

```diff
@@ -284,9 +286,13 @@ void MicroBitLog::_clear(bool fullErase)
 
     // Generate and write FS metadata
     memcpy(metaData.version, MICROBIT_LOG_VERSION, 18);
-    memcpy(metaData.dataStart, "0x00000000\n", 11);
-    memcpy(metaData.logEnd, "0x00000000\n", 11);
+    memcpy(metaData.dataStart, "0x00000000\0", 11);
+    memcpy(metaData.logEnd, "0x00000000\0", 11);
```

Comment:
In flash these are actually stored as null terminated (via `writeNum()`), so I've changed it here as well to make that clear (and the struct comments as well).